lib/controllers/secrets.js

```diff
@@ -0,0 +1,11 @@
+const { Router } = require('express');
+const Secret = require('../models/Secrets.js');
+
+module.exports = Router().get('/', async (req, res, next) => {
```

Comment:
since you're not using the authenticate middleware here, it's not restricting access to the secrets people who are signed in